src/option-parser.h

```diff
@@ -17,47 +17,83 @@
 #ifndef WABT_OPTION_PARSER_H_
 #define WABT_OPTION_PARSER_H_
 
+#include <functional>
+#include <string>
+#include <vector>
+
 #include "common.h"
 
 namespace wabt {
 
-enum class HasArgument {
-  No = 0,
-  Yes = 1,
-};
+class OptionParser {
+ public:
+  enum class HasArgument { No, Yes };
```

Comment:
If you keep the explicit = 0 and = 1 can you continue to write "if (has_argument)"?  Seem a bit clunky to have to explicity compare to something called 'Yes' for variables like has_argument which are clearly boolean already.